### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by avinashkrishna07**
Added guidelines and rules for contributing to the repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## GUIDELINES FOR CONTRIBUTING.
+
+- Python is the preferred programming language for contribution.
+- Either fix the codes that are mentioned on issues or create a new issue to make ONE program that solves a specific task and create a pull request.
+- Use smallercase letters for filename and don't include spaces in the filename.
+- Use proper file extensions for your file.
+- Don't Copy paste codes from the internet as it will be considered as invalid.
+- Create a new Issue stating what the program does,and a short description of the program if you want to add new programs that does a specific task into the repo.
+- Happy Coding :)
```
Comment (rohan472000) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by rohan472000 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (avinashkrishna07) [reply CONTRIBUTING.md]: Updated the CONTRIBUTING.md.

**Review by rohan472000 — APPROVED** [review]

**Comment by rohan472000** [comment]
Thanks for your contribution, if interested then you can work on PR template of this repo which should appear while raising PR.

**Comment by avinashkrishna07** [comment]
Okay, thank you so much @rohan472000  for giving me oppurtunity to contribute. Actually I'm beginner
 in Open-Source contributions. So definitely I'll contribute if i could.

**Pull request merged**